text/0014-export.md

```diff
@@ -0,0 +1,125 @@
+- Feature Name: Export Dataset
+- Start Date: 2018-08-27
+- RFC PR: <!-- (leave this empty) -->
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+Peers need to be able to get q
+
+# Motivation
+[motivation]: #motivation 
+
+<!-- Why are we doing this? What use cases does it support? What is the expected outcome? -->
+In order for Qri to be useful to a large swath of folks, we need a well 
+defined way to get datasets out of Qri. Our two methods for doing so are the
+`/export` api endpoint and the `qri export` command. Both methods should have
+the same options, level of control, and outputs.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+<!-- Explain the proposal as if it was already included in the language and you were teaching it to a Qri _developer_. That generally means:
+
+- Introducing new named concepts.
+- Explaining the feature largely in terms of examples.
+- Explaining how Qri developer should *think* about the feature, and how it should impact the way they use Qri. It should explain the impact as concretely as possible.
+- If applicable, provide sample error messages, deprecation warnings, or migration guidance.
+- If applicable, describe the differences between teaching this to a Qri developer vs a Qri _user_.
+
+For implementation-oriented RFCs (e.g. for Qri codebase internals), this section should focus on how contributors should think about the change, and give examples of its concrete impact. For policy RFCs, this section should provide an example-driven introduction to the policy, and explain its impact in concrete terms. -->
+Export should allow users to export one, some, or all of:
+  - header (meta, structure, commit)
+  - body, or a section of it
+  - transform
+  - viz
+  - (rendered version of dataset in html?)
+
```

Comment:
not sure if we should allow meta, structure, and commit to be exported as separate files